### PR TITLE
Co.luoja.telyn

### PR DESCRIPTION
**Pull request by Kiwijane3**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Review by hfiguiere — CHANGES_REQUESTED** [review]
From what I can gather, this is x86_64 so you need to disable the `aarch64` build.

Or better, build from source.

**Comment by hfiguiere** [comment]
do you submit the .desktop, .appdata and icon files upstream?

**Comment by hfiguiere** [comment]
Also I'm not even sure you need the GNOME runtime. The freedesktop runtime should be enough at that point.

**Comment by hfiguiere** [comment]
bot, build co.luoja.Telyn

File: co.luoja.Telyn.appdata.xml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+		 Telyn is an application for creating and using parametric sewing patterns.
+		 It supports graphical editing, making the application approachable and easy to use,
+		 and supports the use of templates for improved productivity.
+		 Patterns made with Telyn can automatically adjust to respond to a user's individual measurements, 
+		 enabling a better fit than is possible with standard sizes.
+		 Telyn has flexible support for printing, with the ability to print to any paper size.
+		 </p>
+	</description>
+	<content_rating type="oars-1.1" />
+	<update_contact>janef0421@icloud.com</update_contact>
```
Comment (hfiguiere) [inline co.luoja.Telyn.appdata.xml:27]: missing: `<releases>`, `<screenshots>`

File: co.luoja.Telyn.yml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+      # Place the shared object files directly in /app/lib
+      - cp -r swift-5.5-RELEASE-ubuntu20.04/usr/lib/swift/linux/* /app/lib
+  - name: telyngtk
+    buildsystem: simple
+    sources:
+      - type: archive
+        url: https://gitlab.com/t2056/telyngtk/-/jobs/1639790825/artifacts/raw/TelynGtk-Release.tar.gz
+        sha256: a26ddbad02bfb4f17da6fddac1eba4c35122bed148b83b567a9bdf4956c3f0aa
+    build-commands:
+      - ls
```
Comment (hfiguiere) [inline co.luoja.Telyn.yml:33]:
```suggestion
```

**Comment by Kiwijane3** [comment]
The resources are now merged upstream.

**Review by hfiguiere — CHANGES_REQUESTED** [review]

**Comment by Kiwijane3** [comment]
I have updated the appdata to include images and releases, and added a flathub.json specifying x86_64 builds.

**Comment by hfiguiere** [comment]
bot, build co.luoja.Telyn

File: co.luoja.Telyn.appdata.xml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+		 It supports graphical editing, making the application approachable and easy to use,
+		 and supports the use of templates for improved productivity.
+		 Patterns made with Telyn can automatically adjust to respond to a user's individual measurements, 
+		 enabling a better fit than is possible with standard sizes.
+		 Telyn has flexible support for printing, with the ability to print to any paper size.
+		 </p>
+	</description>
+	<content_rating type="oars-1.1" />
+	<releases> 
+		<release version="1.0.1" date="2021-10-01"
```
Comment (hfiguiere) [inline co.luoja.Telyn.appdata.xml:38]:
```suggestion
		<release version="1.0.1" date="2021-10-01" />
```

**Comment by Kiwijane3** [comment]
I have updated the appdata file, it should now validate.

**Comment by hfiguiere** [comment]
bot, build co.luoja.Telyn

**Comment by Kiwijane3** [comment]
There seems to be some bugs. I'll iron those out.

**Comment by Kiwijane3** [comment]
I've resolved the bugs I detected and pushed a modified manifest that fixes them. The application should now be ready to publish.

File: co.luoja.Telyn.yml [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+    sources:
+      - type: git
+        url: https://gitlab.com/t2056/telyngtk.git
+        tag: 1.0.3     
+        commit: 98210f26ffa9f1c22bb3b9d4da42290f3170f8d3
+    build-commands:
+      - install -D co.luoja.Telyn.desktop /app/share/applications/co.luoja.Telyn.desktop
+      - install -D co.luoja.Telyn.svg /app/share/icons/hicolor/scalable/apps/co.luoja.Telyn.svg
+      - install -D co.luoja.Telyn-symbolic.svg /app/share/icons/hicolor/scalable/apps/co.luoja.Telyn-symbolic.svg
+      - install -D co.luoja.Telyn.appdata.xml /app/share/metainfo/co.luoja.Telyn.appdata.xml
```
Comment (barthalion) [inline co.luoja.Telyn.yml:48]: It would probably make sense to include these files in the tar.gz you are using above.

**Comment by barthalion** [comment]
bot, build co.luoja.Telyn

**Pull request closed without merging**